src/parse.rs

```diff
@@ -163,7 +163,7 @@ fn illegal(parser: &mut Parser, banned: &str) {
     parser.span_fatal(span, format!("Expected one of: `{}`, but found: `{}`",
         format!("{}, {}, {}, {}, {}, {}, {}, {}",
                 BEFORE_EACH, AFTER_EACH, BEFORE, AFTER,
-                IT, BENCH, FAILING, DESCRIBE).as_slice(),
-        banned).as_slice());
+                IT, BENCH, FAILING, DESCRIBE),
+        banned).as_ref());
```

Comment:
I'd rather use deref coercions through `&format!(...)` vs. `format!().as_ref()`. Applies in a few places.